src/js/view/view.js

```diff
@@ -217,9 +217,9 @@ function View(_api, _model) {
         focusHelper = new UI(_playerElement).on('click', function() {});
         fullscreenHelpers = requestFullscreenHelper(_playerElement, document, _fullscreenChangeHandler);
 
-        if (_floatOnScroll) {
+        if (_floatOnScroll && _floatOnScroll.dismissible !== false) {
             const floatCloseButton = new FloatingCloseButton(_wrapperElement.querySelector('.jw-top'));
-            floatCloseButton.setup(() => _stopFloating(true), _localization.close);
+            floatCloseButton.setup(() => this.stopFloating(true, true), _localization.close);
```

Comment:
```
floatCloseButton.setup(() => { 
    this.stopFloating(true);
     _api.pause({ reason: 'interaction' }); 
    }, _localization.close); 
```

This would be doing the same thing, right?